tests/test_utils.py

```diff
@@ -23,23 +23,28 @@ def setUp(self):
     def tearDown(self):
         supported_models.cache_clear()
 
-    def test_supported_models(self):
-        models = ["Foo", "Bar"]
-        app_config = Mock(
-            spec=[], cms_extension=Mock(spec=[], navigation_apps_models=models)
-        )
-        with patch.object(apps, "get_app_config", return_value=app_config):
-            self.assertEqual(supported_models(), models)
+    def test_supported_models_with_installed_apps(self):
+        """Tests that all models registered in cms_config shows
```

Comment:
Try not to start a test with: "Tests that", we all do it and this comment was said to me: "It's obvious that it's testing that because it's a test"

You can list the business / technical requirement or edge case:
"All models registered in cms_config are shown as supported models"